web/src/components/molecules/Content/ContentTable/index.tsx

```diff
@@ -1,10 +1,12 @@
 import ProTable, { ListToolBarProps, ProColumns } from "@reearth-cms/components/atoms/ProTable";
+import { Item } from "@reearth-cms/components/molecules/Content/types";
 
-const columns: ProColumns<any>[] = [];
-
-const ContentTable: React.FC = () => {
-  const dataSource: [] = [];
+export type Props = {
+  contentTableFields?: any[];
```

Comment:
As always, can't you type this better than `any`?

Reply:
@KaWaite , Backend typed it as `any`, it can be any type from string to an integer to array...etc
![1](https://user-images.githubusercontent.com/78056580/194227264-03d5eadc-e85c-439f-a3bb-cf36d0f77857.png)

Reply:
Fixed in all places now.